rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -655,6 +655,78 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     errorLog.readAndClearErrorVector.unsafeRunSync should be(Vector.empty[InterpreterError])
   }
 
+  "let x = 1, y = 2 in { @\"result\"!(x, y) }" should "return (1, 2) to \"result\"" in {
+    implicit val errorLog = new ErrorLog[Task]()
+    val splitRand         = rand.splitByte(0)
+    val result = withTestSpace(errorLog) {
+      case TestFixture(space, reducer) =>
+        val pattern0 = EVar(FreeVar(0))
+        val target0  = GInt(1)
+        val pattern1 = EVar(FreeVar(0))
+        val target1  = GInt(2)
+        val matchTerm = Let(
+          Seq(
+            Declaration(pattern0, target0, freeCount = 1),
+            Declaration(pattern1, target1, freeCount = 1)
+          ),
+          Send(
+            GString("result"),
+            List(EVar(BoundVar(1)), EVar(BoundVar(0))),
+            false,
+            BitSet(0, 1)
+          ),
+          2,
+          BitSet()
+        )
+        val matchTask = reducer.eval(matchTerm)(Env(), splitRand)
+        val inspectTask = for {
+          _   <- matchTask
+          res <- space.toMap
+        } yield res
+        Await.result(inspectTask.runToFuture, 3.seconds)
+    }
+    val channel: Par = GString("result")
+
+    checkData(result)(channel, Seq(GInt(1), GInt(2)), splitRand)
```

Comment:
Could you refrain from using `checkData`? It makes finding the failure, should it occur, much harder.
Check out the peek test in this file (after rebasing with dev)